### DISCUSSION TITLE
中文app名称打包不成功

### DISCUSSION THREAD

**Discussion by gegj** [tw93/Pake#429] [Q&A] [answered]
Name 设置字母+中文 必失败，能不能安装目录单独设置，不调用Name的名称。Fork 最新的版本用打包的。用全字母就可以打包成功。

**Comment by Tlntin** [comment, marked as answer]
是的，不支持中文。
这个不是bug，是底层不支持，因为最终生成的运行文件，是英文。比如windows下面的微信，运行文件是wechat.exe而不是“微信.exe”。
当然MacOS貌似可以，Linux/Windows是不支持中文名作为包名的。
最后，还是建议别用中文做包名，因为会有各种编码问题，导致最终名称乱码。
